graf2d/gpadv7/inc/ROOT/RAttrBorder.hxx

```diff
@@ -6,39 +6,40 @@
  * For the list of contributors see $ROOTSYS/README/CREDITS.             *
  *************************************************************************/
 
-#ifndef ROOT7_RAttrOnFrame
-#define ROOT7_RAttrOnFrame
+#ifndef ROOT7_RAttrBorder
+#define ROOT7_RAttrBorder
 
+#include <ROOT/RAttrLine.hxx>
 #include <ROOT/RAttrValue.hxx>
 
 namespace ROOT {
 namespace Experimental {
 
-/** \class RAttrOnFrame
+/** \class RAttrBorder
 \ingroup GpadROOT7
 \author Sergey Linev <s.linev@gsi.de>
-\date 2021-05-06
-\brief Class which add onframe property for drawable, used when drawable can be drawn on frame or outside frame
+\date 2021-06-08
+\brief Drawing line attributes for different objects.
 \warning This is part of the ROOT 7 prototype! It will change without notice. It might trigger earthquakes. Feedback is welcome!
 */
 
-class RAttrOnFrame {
+class RAttrBorder : public RAttrLine {
 
-   RAttrValue<bool> fOnFrame;    ///<! is drawable drawn on the frame or not
-   RAttrValue<bool> fClipping;   ///<! if drawable should be clipped by frame
+   RAttrValue<int>     fRx{this, "rx", 0};              ///<! rounding on x coordinate, px
+   RAttrValue<int>     fRy{this, "ry", 0};              ///<! rounding on y coordinate, px
 
-public:
+   R__ATTR_CLASS(RAttrBorder, "border");
```

Comment:
Is this public? If so I'd appreciate an explicit
```suggestion
public:
   R__ATTR_CLASS(RAttrBorder, "border");
```

Reply:
Macro `R__ATTR_CLASS` has protected and public methods and define visibility scope itself

Reply:
OK but then we *must* have a subsequent `public:` or `private:` - we cannot rely on readers of the class to know what the access is after `R__ATTR_CLASS`. IIRC that was missing?